change_notes/2024-01-26-exclusion-a5-0-2.md

```diff
@@ -0,0 +1,2 @@
+`A5-0-2`: `cpp/autosar/non-boolean-if-condition`, `cpp/autosar/non-boolean-iteration-condition`
```

Comment:
```suggestion
`A5-0-2` - `NonBooleanIfStmt.qll`, `NonBooleanIterationStmt.qll`:
```

Adjust the format of the header to match that of what is specified in the developer handbook,